acceptance/README.md

```diff
@@ -0,0 +1,114 @@
+# Acceptance tests for puppetlabs-puppet_agent
+
+## Background
+
+### About Beaker
+
+Beaker is a host provisioning and an acceptance testing framework. If you are
+unfamiliar with beaker, you can start with these documents:
+
+- [The Beaker DSL document](https://github.com/puppetlabs/beaker/blob/master/docs/how_to/the_beaker_dsl.md) will help you understand the test code in the `tests/` and `pre_suite/` subdirectories.
+- [The Beaker Style Guide](https://github.com/puppetlabs/beaker/blob/master/docs/concepts/style_guide.md) will help you write new test code.
+- [Argument Processing](https://github.com/puppetlabs/beaker/blob/master/docs/concepts/argument_processing_and_precedence.md) and [Using Subcommands](https://github.com/puppetlabs/beaker/blob/master/docs/tutorials/subcommands.md) have more information on beaker's command line and environmental options.
+
+### About these tests
+
+This module is responsible for upgrading and downgrading puppet-agent.  Testing
+this behavior necessarily involves repeatedly installing and uninstalling
+puppet-agent. Ideally, the test hosts would be totally destroyed and
+reprovisioned before each fresh install of puppet-agent, but beaker does not
+support workflows like this. Instead:
+
+- Use the `run_setup` helper method to install puppet-agent, plus this module
+  and its dependencies, before the main content of each of your tests.
+- Use the `run_teardown` helper method to uninstall puppet-agent and remove
+  modules in your tests' teardown steps.
+
+See [helpers.rb](./helpers.rb) for the impelementations of these methods.
+
+#### Environment variables affecting test behavior
+
+When the `run_setup` helper installs puppet-agent, it determines the version to
+install as follows:
+
+1. If the `FROM_AGENT_VERSION` environment variable is set, install that version.
+2. Otherwise, install the latest agent from the `FROM_PUPPET_COLLECTION`
```

Comment:
Didn't we agree to remove the `PUPPET_COLLECTION` env variables and just stick with package versions for now? (This was before Christmas). I think that was @branan's suggestion.